armsrc/appmain.c

```diff
@@ -1419,16 +1437,22 @@ void  __attribute__((noreturn)) AppMain(void) {
 	StartTickCount();
   	
 #ifdef WITH_LCD
-	LCDInit();
+//	LCDInit();
 #endif
 
 #ifdef WITH_SMARTCARD
-	I2C_init();
+//	I2C_init();
```

Comment:
Is there a specific reason for not initializing smart card module  here?

Reply:
Merge conflict with old version from iceman fork.